k2/csrc/array.h

```diff
@@ -66,14 +66,15 @@ class Array1 {
     Init(ctx, size);
 
     // TODO(haowen): there's no such definition
-    // `Eval(ContextPtr, T*, int, Callable&)` now
+    // `Eval(ContextPtr, T*, int32_t, Callable&)` now
     Eval(ctx, Data(), size, std::forward<Callable>(callable));
   }
   */
 
   Array1(ContextPtr ctx, int32_t size) { Init(ctx, size); }
 
   // Creates an array that is not valid, e.g. you cannot call Context() on it.
+  // TODO(haowen): why do we need this version?
```

Comment:
I think I delete this line in my last PR (or the one before that), as we need such default constructor in ragged-related algorithms.

Reply:
deleted